lib/redux/state/app_state.dart

```diff
@@ -170,6 +176,9 @@ class GlobalState {
       apdexBeforeByProjectId: apdexBeforeByProjectId ?? this.apdexBeforeByProjectId,
       issuesByProjectSlug: issuesByProjectSlug ?? this.issuesByProjectSlug,
       selectedOrganization: selectedOrganization ?? this.selectedOrganization,
+      isLoading: isLoading ?? this.isLoading,
+      loadingText: loadingText ?? this.loadingText,
```

Comment:
Since `loadingText ` and `loadingProgress` are nullable, we can never reset then back to null once they have a value. They will always fallback to the current instance value. See `setTokenNull` for reference.

Reply:
I would just keep the `boolean` so we don't need to deal with the "nullability" of text and progress. Only when the boolean is true we show the loading screen.

Reply:
We should still set the correct state when we are done, as we leave the old text and never reset it. Could become an issue if we need to show loading again where we just set the boolean, then we would show old text. Better to do the correct thing in the reducer.

Reply:
We can reset the values if loading is false for example. `loadingText: !isLoading ? null : loadingText ?? this.loadingText,`